scripts/bash/run_enceladus.sh

```diff
@@ -205,16 +209,19 @@ add_to_cmd_line "--driver-memory" ${DRIVER_MEMORY}
 CMD_LINE="$CMD_LINE --conf \"$CONF\" --class $CLASS $JAR"
 
 # Adding command line parameters that go AFTER the jar file
+add_to_cmd_line "--menas-auth-keytab"  ${MENAS_AUTH_KEYTAB}
```

Comment:
This doesn't do a thing but there is one more space than needed.
```suggestion
add_to_cmd_line "--menas-auth-keytab" ${MENAS_AUTH_KEYTAB}
```